lib/SIL/SILFunctionType.cpp

```diff
@@ -290,7 +290,7 @@ CanSILFunctionType SILFunctionType::getAutoDiffAssociatedFunctionType(
               getAssociatedType(param.getType(), "CotangentVector")));
     auto pullbackType = SILFunctionType::get(getGenericSignature(),
                                              ExtInfo(), SILCoroutineKind::None,
-                                             ParameterConvention::Direct_Owned,
+                                         ParameterConvention::Direct_Guaranteed,
```

Comment:
this has become weirdly indented

Reply:
This is to fit within 80 columns. It's a pretty common, but weird, pattern within this code base.

Reply:
Re-formatted.